common/config/babel/.babelrc.js

```diff
@@ -7,12 +7,34 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
       match: '@conditional-compile-remove-from(stable)',
       // A list of features recognized by the conditional compilation preprocessing plugin.
       // "demo" is a special feature, used for demo purposes. For this feature,
-      // The plugin removes any AST node that is preceded by a comment that contains with the tag:
+      // The plugin removes any AST node that is preceded by a comment that contains the tag:
       // @conditional-compile-remove(demo)
       features: [
+        'call-with-chat-composite',
```

Comment:
Food for thought: Now that this is so obviously -- we need to reduce the number of beta-only features.
And also need to know what the criteria are for each feature to be stabilized.

Reply:
@alkwa-msft

Reply:
Agree, there are too many beta only features - as what we mentioned in our chat channel - some of them could be directly developed in main branch without conditional compilation, some could be stabilized already
Conditional compilation is a powerful tool, but overuse it will be a disaster

Reply:
My previous thoughts about what should be conditional compiled:
https://teams.microsoft.com/l/message/19:5e150d250f724d1089c949305efdc8cc@thread.skype/1644444493297?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e9c1fc3-39df-4486-a26a-456d80e80f82&parentMessageId=1644275183011&teamName=Azure%20Communication%20Services&channelName=Web%20UI%20-%20Team&createdTime=1644444493297